model/src/pyrenew/latent/infection_initialization_method.py

```diff
@@ -177,6 +177,7 @@ def initialize_infections(self, I_pre_init: ArrayLike):
                 f"I_pre_init must be an array of size 1. Got size {I_pre_init.size}."
             )
         (rate,) = self.rate()
+        rate = rate.value
```

Comment:
```suggestion
        rate = self.rate()[0].value
```

Reply:
Longer term this should be refactored to follow the [variable naming conventions](https://github.com/CDCgov/multisignal-epi-inference/blob/main/docs/source/developer_documentation.rst#pyrenew-principles), which would allow us to do something like:

```python
rate = self.rate_rv()[0].value
```